src/main/java/com/qaprosoft/carina/demo/guipractice/components/NavigateMenu.java

```diff
@@ -11,39 +11,39 @@
 
 public class NavigateMenu extends AbstractUIObject {
```

Comment:
In your testcase #2 (google doc) you need to verify that "signup" and "login" buttons are not shown. So, we need to create such methods here (for example `isLoginButtonPresent`)
In your testcase #3 you need to verify "Welcome ***username***" label. Create method for this

Reply:
For verifying if these buttons are shown, can I just edit the excisting method like this:

	public void clickLogoutButton() {
		logoutButton.clickIfPresent();
	}

I saw it on carina-demo example (WeValuePrivacyAd).